pkg/chain_iterator/block_batch_iterator.go

```diff
@@ -65,6 +67,7 @@ type BlockBatchIteratorConfig struct {
 	OnBlocks              OnBlocksFunc
 	ReorgRewindDepth      *uint64
 	RetryInterval         time.Duration
+	BlockConfirmations    *uint64
```

Comment:
ditto

Reply:
It is not necessary to add this in BlockBatchIteratorConfig. There is no use in this struct, do I still need to add it to this struct?